common/history/pops/07_south_america.txt

```diff
@@ -722,14 +722,18 @@
 		}
 	}
 	s:STATE_COTE_DU_CAFE = { #these pops will need to be reworked if/when Atlantia is added
```

Comment:
you can probably remove my old comment now. Also, on line 577, you'll want to add in the Atlantian pops for Nouvelle Alsace.